index.html

```diff
@@ -4854,8 +4854,8 @@ <h3>Language and Base Direction</h3>
       </pre>
 
       <p>
-An example utilizing the design pattern above is shown below to express the
-title of a book in the English language without specifying a text direction:
+Utilizing the design pattern above the following example expresses the title of
```

Comment:
`Utilizing the design pattern above the following`
-> `Using the design pattern above, the following`

Reply:
Done.